README.md

```diff
@@ -88,34 +108,52 @@ yum install -y epel-release
 yum install -y readline-devel leveldb-devel libffi-devel gcc-c++ redhat-rpm-config gcc python-devel openssl-devel
 ```
 
-### For all of these, make sure that the `Chains` directory in your project has the proper write permissions
+#### Windows
 
-##### Windows
+Help needed. Installing the Python package plyvel seems to require C++ compiler
+support tied to Visual Studio and libraries.
 
-Not sure. Installing the Python package plyvel seems to require C++ compiler support tied to Visual Studio and libraries.
+### Python 3.5
 
--------------------
+neo-python is currently only compatible with **Python 3.5** (due to its `byteplay3` dependency).
 
-### Virtual Environment
+On *nix systems, install Python 3.5 via some package manager, or download an installation package
```

Comment:
say "via your package manager" (or homebrew on OSX).

Reply:
According to https://discourse.brew.sh/t/how-do-i-install-python-3-5-instead-of-3-6/675 Homebrew is not an option (I didn't manage to get it installed via brew)

Reply:
Correction, not an **easy** option.

Reply:
Alright, good point. leave homebrew out then ;) 

Get on Discord and say hello btw.